src/components/sidebar/sidebar.style.js

```diff
@@ -35,7 +35,7 @@ const SidebarStyle = styled.div`
       ${position}: 0;
     `};
 
-  ${StyledIconButton} {
+  > ${StyledIconButton}:first-of-type {
```

Comment:
I'm not sure this will fix it if no `onCancel` prop is passed then the style will apply to the first `StyledIconButton`